awx/ui_next/jest.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     '\\.(css|scss|less)$': '<rootDir>/__mocks__/styleMock.js',
     '^@api(.*)$': '<rootDir>/src/api$1',
     '^@components(.*)$': '<rootDir>/src/components$1',
+    '@constants$': '<rootDir>/src/constants.js',
```

Comment:
Generally we want to keep this in sync with https://github.com/ansible/awx/blob/devel/awx/ui_next/webpack.config.js#L61-L67 and vice-versa.

Admittedly, our configuration pattern for aliases isn't very DRY but we don't add or remove them very often.

Reply:
nm - there it is!